enzyme/Enzyme/Utils.cpp

```diff
@@ -2511,9 +2548,18 @@ llvm::Value *is_normal(IRBuilder<> &B, llvm::Value *trans, bool byRef) {
 // However, if we ask openBlas c ABI,
 // it is one of the following 32 bit integers values:
 // enum CBLAS_TRANSPOSE {CblasNoTrans=111, CblasTrans=112, CblasConjTrans=113};
-llvm::Value *transpose(IRBuilder<> &B, llvm::Value *V) {
+llvm::Value *transpose(IRBuilder<> &B, llvm::Value *V, bool cublas) {
   llvm::Type *T = V->getType();
   Value *out;
+  if (cublas) {
```

Comment:
@wsmoses 
This relies on the integer representation of enums in c++, I took the order from some docs, is there a nicer way?

Reply:
We can write our own enum or constant variables Eg Cublas_N to make it nicer, but unfortunately otherwise not. We just need to keep the ABI the same and we don't want to require cuBLAS to compile enzyme.